src/test/logic/calculate.test.js

```diff
@@ -0,0 +1,72 @@
+import Calculate from '../../logic/calculate';
```

Comment:
- [ ] In this file please add tests for all your calculate functionalities like: %, +/-, ".", AC, and the division by 0.